kubernetes/samples/charts/util/setupLoadBalancer.sh

```diff
@@ -13,6 +13,7 @@ UTILDIR="$(dirname "$(readlink -f "$0")")"
 #Kubernetes command line interface. 
 #Default is 'kubectl' if KUBERNETES_CLI env variable is not set.  
 kubernetesCli=${KUBERNETES_CLI:-kubectl}
+image_pull_policy="IfNotPresent"
```

Comment:
Since this is hard coded so we need the variable ?

Reply:
removed